docs/src/main/asciidoc/security-basic-authentication-concept.adoc

```diff
@@ -1,10 +1,11 @@
 [id="security-basic-authentication-concept"]
-= Basic authentication
+= Basic authentication and JPA
```

Comment:
```suggestion
= Basic authentication
```

Reply:
@sberyozkin - If a Quarkus customer will typically implement Basic authentication with JPA, should this concept include JPA in the topic title and a sub-section? 

We introduce JPA in the tutorial but it's not mentioned in any Quarkus concept topic. We explain all the other supported authentication methods/protocols. Or does JPA in Quarkus deserve its own concept topic?
Thanks